themes/startwords/layouts/article/single.html

```diff
@@ -14,11 +33,13 @@
         <h1>{{ .Params.Title }}</h1>
         {{ partial "authors" . }}
         {{ partial "pubdate" . }}
+        {{ if .Params.doi }}
+         <a href="http://doi.org/{{ .Params.doi }}" rel="alternate">doi:{{ .Params.doi }}</a>
```

Comment:
if you throw a `class="doi"` on this link it will use the right style, which adds the `doi:` text in a `::before` so you don't have to insert it into the link text.

should we use `rel="alternate"` for the DOI links to articles on the homepage/issue page also, or does it only make sense when viewing the article itself?

Reply:
Cool, thanks — I thought about adding a doi class for later styling, forgot it was already meaningful with the prefix defined elsewhere.

I think `rel="alternate"` should only be used on the page for the article itself, since the relation is relative to the current url.